dev/tests/integration/testsuite/Magento/Framework/App/Cache/Frontend/PoolTest.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Framework\App\Cache\Frontend;
+
+use Magento\Framework\ObjectManager\ConfigInterface as ObjectManagerConfig;
+use Magento\TestFramework\ObjectManager;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * This is prime example for a senseless comment that only exists to satisfy the coding standard.
```

Comment:
I feel your pain, this is one of mine top priorities in phpcs-related stuff, annoys me since `@return void` was introduced.

Reply:
unfortunately Magento code is there for everybody to see and interpret and we have to do yet better, meaningful, rhetorical comments

Reply:
Thankfully it's no longer necessary to add useless comments due to the updated phpdoc coding style guide!
I'll remove the docblock.